assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -87,6 +95,28 @@ export default function ReminderBanner( {
 		ACTIVATION_ACKNOWLEDGEMENT_TOOLTIP_STATE_KEY
 	);
 
+	const viewContext = useViewContext();
+	const eventCategory = `${ viewContext }_ga4-reminder-notification`;
+
+	useMount( () => {
+		trackEvent( eventCategory, 'view_notification' );
+	} );
```

Comment:
I [mentioned this in the IB review](https://github.com/google/site-kit-wp/issues/6042#issuecomment-1289763682), but looking at the code and the ACs I'm not quite clear on it… I think this isn't quite right; this component will output a few different things based on the state of the app, and I'm not sure we want to send a view notification for every one of them.

I think we really only want to consider the banner "viewed" if this component is output: https://github.com/google/site-kit-wp/blob/7a77e8f06e2ccb040213149d1e1450ec38e34b3c/assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js#L253

Can you confirm this @felixarntz? It's a bit unclear from the ACs if we should send that event anytime any part of this component is output or only when the actual `BannerNotification` is shown.

Reply:
Looks like the `view_notification` should only send when this component is rendered. See @felixarntz's response here: https://github.com/google/site-kit-wp/pull/6096#discussion_r1012246885